pykeepass/group.py

```diff
@@ -9,22 +9,24 @@
 
 class Group(BaseElement):
 
-    def __init__(self, name=None, element=None, icon=None, notes=None,
-                 version=None, expires=None, expiry_time=None):
+    def __init__(self, name=None, element=None, icon=None, customicon=None, notes=None,
+                 version=None, expires=None, expiry_time=None, meta=None):
 
         assert type(version) is tuple, 'The provided version is not a tuple, but a {}'.format(
             type(version)
         )
         self._version = version
-
+        self._meta = meta
```

Comment:
Just curious: what's this `meta` stuff?

Reply:
The KDBX XML contains a Meta tag which contains the customicons (the actual image files). Since it doesn't use a generic key index but instead a base64 index to link the image to the entry, I send the meta tag + children to the the entry, so the base64 index can be converted to a numerical one (as the keepass UI does as well)